src/main/java/org/folio/clients/impl/SourceRecordStorageClient.java

```diff
@@ -0,0 +1,74 @@
+package org.folio.clients.impl;
+
+import io.vertx.core.json.JsonObject;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpRequestBase;
+import org.apache.http.util.EntityUtils;
+import org.folio.clients.SynchronousOkapiClient;
+import org.folio.util.OkapiConnectionParams;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.lang.invoke.MethodHandles;
+import java.net.URI;
+import java.net.URLEncoder;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * The client that synchronously communicates with the Source Record Storage(SRS).
+ * <p>
+ * Retrieves collection of underlying SRS records as a source of truth.
+ * Retrieves collection of Inventory records that do not have underlying SRS records
+ *
+ * @link https://github.com/folio-org/mod-source-record-storage
+ */
+@Component
+public class SourceRecordStorageClient extends SynchronousOkapiClient {
+
+  private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
LoggerFactory.getLogger(SourceRecordStorageClient.class);

Reply:
Why is this better?